frontend/src/app/documentview/document-view-table.component.html

```diff
@@ -1,42 +1,47 @@
 <table mat-table [dataSource]="dataSource" matSort class="mat-elevation-z8">
 
-     <ng-container matColumnDef="name">
-      <th mat-header-cell *matHeaderCellDef mat-sort-header> Filename </th>
-      <td mat-cell *matCellDef="let element"> {{element.name}} </td>
-    </ng-container>
-  
-    <ng-container matColumnDef="path">
-      <th mat-header-cell *matHeaderCellDef mat-sort-header> Path </th>
-      <td mat-cell *matCellDef="let element"> {{element.path}} </td>
-    </ng-container>
-  
-    <ng-container matColumnDef="type">
-      <th mat-header-cell *matHeaderCellDef mat-sort-header> Type </th>
-      <td mat-cell *matCellDef="let element"> {{element.type}} </td>
-    </ng-container>
-  
-    <ng-container matColumnDef="lang">
-      <th mat-header-cell *matHeaderCellDef mat-sort-header> Language </th>
-      <td mat-cell *matCellDef="let element"> {{element.lang}} </td>
-    </ng-container>
-  
-    <ng-container matColumnDef="size">
-      <th mat-header-cell *matHeaderCellDef mat-sort-header> Size </th>
-      <td mat-cell *matCellDef="let element"> {{element.size}} </td>
-    </ng-container>
-  
-    <ng-container matColumnDef="createdAt">
-      <th mat-header-cell *matHeaderCellDef mat-sort-header> CreatedAt </th>
-      <td mat-cell *matCellDef="let element"> {{element.createdAt}} </td>
-    </ng-container>
+  <ng-container matColumnDef="name">
+    <th mat-header-cell *matHeaderCellDef mat-sort-header> Filename </th>
+    <td mat-cell *matCellDef="let element"> {{element.title}} </td>
+  </ng-container>
 
-    <ng-container matColumnDef="MyTags">
-      <th mat-header-cell *matHeaderCellDef> MyTags </th>
-      <td mat-cell *matCellDef="let element"> {{element.tags}} </td>
-    </ng-container>
-    
-    <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
-    <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
-</table>
-  
-  
\ No newline at end of file
+  <ng-container matColumnDef="type">
+    <th mat-header-cell *matHeaderCellDef mat-sort-header> Type </th>
+    <td mat-cell *matCellDef="let element"> {{element.type}} </td>
+  </ng-container>
+
+  <ng-container matColumnDef="lang">
+    <th mat-header-cell *matHeaderCellDef mat-sort-header> Language </th>
+    <td mat-cell *matCellDef="let element"> {{element.language}} </td>
+  </ng-container>
+
+  <ng-container matColumnDef="size">
+    <th mat-header-cell *matHeaderCellDef mat-sort-header> Size </th>
+    <td mat-cell *matCellDef="let element"> {{element.size}} </td>
+  </ng-container>
+
+  <ng-container matColumnDef="createdAt">
+    <th mat-header-cell *matHeaderCellDef mat-sort-header> CreatedAt </th>
+    <td mat-cell *matCellDef="let element"> {{element.creation_date | date:'short'}} </td>
+  </ng-container>
```

Comment:
The matColumnDef needs to be named the same as the variable used in mat-cell or sorting doesnt work. e.g. matColumnDef="lang" needs to be matColumnDef="language"